### DISCUSSION TITLE
What the difference between scopes?

### DISCUSSION THREAD

**Discussion by siranweb** [exuanbo/di-wise#5] [Q&A] [answered]
Hi! First of all, thank you for this awesome package 🔥 

There is question. For me, it's not clear what the difference between `Transient`, `Resolution` and `Container` scopes? Can you add more details to README?

**Comment by exuanbo** [comment, marked as answer]
Hi, is this clearer?

https://github.com/exuanbo/di-wise/tree/docs/clarify-scopes?tab=readme-ov-file#various-injection-scopes

**Reply by siranweb** [reply]
Perfect, thanks!
